2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Configure stats pid to 1854932
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Loading settings from /home/vb2386/.config/wandb/settings
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Loading settings from /scratch/vb2386/Documents/nerf-pytorch/wandb/settings
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'profile_nerf.py', 'program_abspath': '/scratch/vb2386/Documents/nerf-pytorch/profile_nerf.py', 'program': 'profile_nerf.py'}
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_init.py:_log_setup():521] Logging user logs to /scratch/vb2386/Documents/nerf-pytorch/wandb/run-20240420_225541-ov3mqcmd/logs/debug.log
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_init.py:_log_setup():522] Logging internal logs to /scratch/vb2386/Documents/nerf-pytorch/wandb/run-20240420_225541-ov3mqcmd/logs/debug-internal.log
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_init.py:init():561] calling init triggers
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_init.py:init():611] starting backend
2024-04-20 22:55:41,629 INFO    MainThread:1854932 [wandb_init.py:init():615] setting up manager
2024-04-20 22:55:41,632 INFO    MainThread:1854932 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-20 22:55:41,634 INFO    MainThread:1854932 [wandb_init.py:init():623] backend started and connected
2024-04-20 22:55:41,636 INFO    MainThread:1854932 [wandb_init.py:init():715] updated telemetry
2024-04-20 22:55:41,665 INFO    MainThread:1854932 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-20 22:55:41,861 INFO    MainThread:1854932 [wandb_run.py:_on_init():2357] communicating current version
2024-04-20 22:55:41,902 INFO    MainThread:1854932 [wandb_run.py:_on_init():2366] got version response 
2024-04-20 22:55:41,902 INFO    MainThread:1854932 [wandb_init.py:init():799] starting run threads in backend
2024-04-20 22:55:42,436 INFO    MainThread:1854932 [wandb_run.py:_console_start():2335] atexit reg
2024-04-20 22:55:42,436 INFO    MainThread:1854932 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-20 22:55:42,436 INFO    MainThread:1854932 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-20 22:55:42,436 INFO    MainThread:1854932 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-20 22:55:42,437 INFO    MainThread:1854932 [wandb_init.py:init():842] run started, returning control to user process
2024-04-20 22:55:42,847 INFO    MainThread:1854932 [pyagent.py:run():324] Starting sweep agent: entity=None, project=None, count=5
2024-04-20 22:55:48,544 WARNING MsgRouterThr:1854932 [router.py:message_loop():77] message_loop has been closed
2024-04-20 22:55:50,416 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-20 22:55:50,416 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Configure stats pid to 1854932
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Loading settings from /home/vb2386/.config/wandb/settings
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Loading settings from /scratch/vb2386/Documents/nerf-pytorch/wandb/settings
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'nerf_sweep', 'entity': 'vb2386', 'root_dir': '/scratch/vb2386/Documents/nerf-pytorch', 'sweep_id': 'yiqiimnb', 'run_id': '7qywxkho', 'sweep_param_path': '/scratch/vb2386/Documents/nerf-pytorch/wandb/sweep-yiqiimnb/config-7qywxkho.yaml'}
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'profile_nerf.py', 'program_abspath': '/scratch/vb2386/Documents/nerf-pytorch/profile_nerf.py', 'program': 'profile_nerf.py'}
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_init.py:_log_setup():521] Logging user logs to /scratch/vb2386/Documents/nerf-pytorch/wandb/run-20240420_225550-7qywxkho/logs/debug.log
2024-04-20 22:55:50,420 INFO    Thread-5  :1854932 [wandb_init.py:_log_setup():522] Logging internal logs to /scratch/vb2386/Documents/nerf-pytorch/wandb/run-20240420_225550-7qywxkho/logs/debug-internal.log
2024-04-20 22:55:50,421 INFO    Thread-5  :1854932 [wandb_init.py:init():561] calling init triggers
2024-04-20 22:55:50,421 INFO    Thread-5  :1854932 [wandb_init.py:init():568] wandb.init called with sweep_config: {'batch_size': 2048, 'chunk': 16384, 'lrate': 0.0005}
config: {}
2024-04-20 22:55:50,421 INFO    Thread-5  :1854932 [wandb_init.py:init():606] wandb.init() called when a run is still active
2024-04-20 22:55:50,478 ERROR   Thread-5  :1854932 [wandb_init.py:init():1209] error
Traceback (most recent call last):
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    run = wi.init()
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 608, in init
    tel.feature.init_return_run = True
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/lib/telemetry.py", line 42, in __exit__
    self._run._telemetry_callback(self._obj)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 763, in _telemetry_callback
    self._telemetry_flush()
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 774, in _telemetry_flush
    self._backend.interface._publish_telemetry(self._telemetry_obj)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 101, in _publish_telemetry
    self._publish(rec)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/scratch/vb2386/Documents/nerf-pytorch/venv/lib/python3.8/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
